app/controllers/idv/how_to_verify_controller.rb

```diff
@@ -40,7 +45,7 @@ def self.step_info
         preconditions: ->(idv_session:, user:) do
           self.enabled?
         end,
-        undo_step: ->(idv_session:, user:) {}, # clear any saved data
+        undo_step: ->(idv_session:, user:) { idv_session.skip_doc_auth = nil },
```

Comment:
we're setting this nil b/c a user didn't make a selection, correct?

Reply:
This is if they go back- we want to unset it.

Reply:
ahh ok, thank you explaining that!